src/main.rs

```diff
@@ -1,5 +1,5 @@
 #![allow(non_snake_case)]
-
+#![deny(unused_must_use)] // Enforce handling of Results, to avoid accidentally hiding errors
```

Comment:
Made not probably checking a Result (and other must_use notated types) a compiler error, such that we cant accidentally hide errors.